test/cypress/integration/reusable/open-dialog-extension-test.js

```diff
@@ -0,0 +1,33 @@
+export function clickToOpenDialogExtension(
+  selector = '[data-test-id="open-dialog-extension-button"]'
+) {
+  cy.get('@extension')
+    .find(selector)
+    .click()
+}
+
+export function checkThatDialogIsOpened(selector = '[data-test-id=cf-ui-modal]') {
+  const dialogTitle = 'My awesome dialog extension'
+  cy.get(selector)
+    .should('exist')
+    .and('contain', dialogTitle)
+}
+
+export function checkThatExtensionInDialogIsRendered(
+  selector = '[data-test-id=cf-ui-modal] iframe',
+  dialogSelector = '[data-test-id="my-dialog-extension"]'
+) {
+  cy.waitForIFrame()
+  cy.get(selector).captureIFrameAs('dialogExtension')
+  cy.get('@dialogExtension')
+    .find(dialogSelector)
+    .should('be.visible')
+}
+
+export function openDialogExtensionTest() {
```

Comment:
Maybe this function can take an argument that will get added to the description in the `it` statement to at least give some more context of what was actually being tested? Might not be necessary since it will be in a `describe`.

Reply:
yeah, maybe it makes sense. I'd like to add more tests to collect problems and nuisances to make an informed refactoring.